src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -498,7 +498,7 @@ export class JupyterNotebookBase implements INotebook {
                 ? this.session.requestExecute(
                       {
                           // Remove the cell marker if we have one.
-                          code: cellMatcher.stripFirstMarker(code),
+                          code: cellMatcher.stripFirstMarker(code.replace(/\r\n/g, '\n')),
```

Comment:
These usages look like the only three usages of session.requestExecute. So should this replace instead go into BaseJupyterSession.requestExecute? That would put it in only one place, and prevent someone else from adding a new usage of requestExecute that re-creates this issue.

Reply:
I don't want to refactor the code, ideally we should have just one.
When the old Jupyter layer goes away we'll automatically have just one.
The other two will be in kernel stuff & i'd like to remove the executeSilently to go through kernel as well (but don't want to introduce a refactoring now)

Hopefully we'll get to some of that in debt week.